pkgs/development/python-modules/uxsim/default.nix

```diff
@@ -46,24 +45,20 @@ buildPythonPackage rec {
     tqdm
   ];
 
-  pythonImportsCheck = ["uxsim"];
-
+  pythonImportsCheck = [ "uxsim" ];
 
   # QT_PLUGIN_PATH is required to be set for the program to produce its images
   # our patch sets it to $NIX_QT_PLUGIN_PATH if QT_PLUGIN_PATH is not set
   # and here we replace this string with the actual path to qt plugins
   postInstall = ''
-    substituteInPlace $out/${python3.sitePackages}/uxsim/__init__.py \
```

Comment:
using `python3` here was causing the python3.12 build failure that I didn't notice